tests/unit/test_logging.py

```diff
@@ -224,36 +222,69 @@ def test_bootstrap_logger(caplog: _pytest.logging.LogCaptureFixture) -> None:
     _exercise_logger(caplog, Logger.get_bootstrap_logger())
 
 
+# Helpers for the test below, to make strings slightly shorter: Rendered Labels...
+RL = render_labels(["foo", "bar"])
+# ... and Rendered Labels with Padding.
+RLP = render_labels(["foo", "bar"]) + '   '
+
+
 @pytest.mark.parametrize(
-    ('key', 'value', 'color', 'level', 'labels', 'expected'),
+    ('key', 'value', 'color', 'level', 'labels', 'labels_padding', 'expected'),
     [
-        ('dummy-key', None, None, 0, None, 'dummy-key'),
-        ('dummy-key', 'dummy-value', None, 0, None, 'dummy-key: dummy-value'),
+        ('dummy-key', None, None, 0, None, 0, 'dummy-key'),
+        ('dummy-key', 'dummy-value', None, 0, None, 0, 'dummy-key: dummy-value'),
         (
             'dummy-key',
             'dummy\nmultiline\nvalue',
             None,
             0,
             None,
+            0,
             'dummy-key:\n    dummy\n    multiline\n    value'),
 
-        ('dummy-key', None, None, 2, None, '        dummy-key'),
-        ('dummy-key', 'dummy-value', None, 2, None, '        dummy-key: dummy-value'),
+        ('dummy-key', None, None, 2, None, 0, '        dummy-key'),
+        ('dummy-key', 'dummy-value', None, 2, None, 0, '        dummy-key: dummy-value'),
         (
             'dummy-key',
             'dummy\nmultiline\nvalue',
             None,
             2,
             None,
-            '        dummy-key:\n            dummy\n            multiline\n            value')
+            0,
+            '        dummy-key:\n            dummy\n            multiline\n            value'),
+        (
+            'dummy-key',
+            'dummy\nmultiline\nvalue',
+            None,
+            2,
+            ['foo', 'bar'],
+            0,
+            f'{RL}         dummy-key:\n{RL}             dummy\n{RL}             multiline\n{RL}             value'),  # noqa: E501
+        (
+            'dummy-key',
+            'dummy\nmultiline\nvalue',
+            None,
+            2,
+            ['foo', 'bar'],
+            # Pad labels to occupy their actual length plus 3 more characters
+            len(RL) + 3,
+            f'{RLP}         dummy-key:\n{RLP}             dummy\n{RLP}             multiline\n{RLP}             value')  # noqa: E501
```

Comment:
This is very hard to read. Can it be changed to something like this? IMO it's more clear.
```suggestion
            '        dummy-key:\n'
            '            dummy\n'
            '            multiline\n'
            '            value'),
        (
            'dummy-key',
            'dummy\nmultiline\nvalue',
            None,
            2,
            ['foo', 'bar'],
            0,
            f'{RL}         dummy-key:\n'
            f'{RL}             dummy\n'
            f'{RL}             multiline\n'
            f'{RL}             value'),
        (
            'dummy-key',
            'dummy\nmultiline\nvalue',
            None,
            2,
            ['foo', 'bar'],
            # Pad labels to occupy their actual length plus 3 more characters
            len(RL) + 3,
            f'{RLP}         dummy-key:\n'
            f'{RLP}             dummy\n'
            f'{RLP}             multiline\n'
            f'{RLP}             value')
```

Reply:
Fixed in 41ccff33.